go-controller/hybrid-overlay/pkg/controller/honode_linux.go

```diff
@@ -0,0 +1,124 @@
+package controller
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/types"
+	houtil "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/util"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+
+	"github.com/vishvananda/netlink"
+	kapi "k8s.io/api/core/v1"
+	listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/klog/v2"
+)
+
+// HONodeController is the node hybrid overlay controller
+type HONodeController struct {
+	kube            kube.Interface
+	machineID       string
+	nodeName        string
+	localNodeCIDR   *net.IPNet
+	localNodeIP     net.IP
+	remoteSubnetMap map[string]string // Maps a remote node to its remote subnet
```

Comment:
`machineID` and `remoteSubnetMap`are not being used?

Reply:
removed